pages/docs/frontend-modules/releasing.en-US.mdx

```diff
@@ -1,17 +1,27 @@
+import { Callout } from "nextra-theme-docs";
+
 # Releasing frontend modules
 
-After meeting a reasonable threshold of commits, or after a set time, you will likely want to publish your software to the NPM registry so that consumers can get your latest stable versions.
+[Releasing](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) frontend modules is a critical part of the development process. After meeting a reasonable threshold of commits, or after a set amount of time, you will likely want to publish your software to the NPM registry so that consumers can install it. This guide will walk you through the process of releasing frontend modules in the O3 ecosystem.
+
+<Callout type="info">
+  We're currently working with a bi-weekly release cadence for O3. This means we aim to publish new versions of our
```

Comment:
Are we going with bi-weekly releases or monthly, as per yesterday's discussion?

Reply:
I understood the discussion to mean that the release happens each fortnight then we have two weeks or so for QA, responding to QA issues, then finally publishing to the demo prod server. Did I get that right?

Reply:
Ahh I understood like, each PR takes at least a week to get reviewed, and another week to do changes so in total, almost two weeks to get merged, so that its better to do a release monthly, but I could be badly mistaken.

Reply:
@NethmiRodrigo The goal of "bi-weekly" releases is to try to have a frequent QA cadence, so that the QA process gives us more rapid feedback. If we do a monthly "release" (and, remember, what we're currently calling "releases" are really "promotions to QA"), then we're likely to be catching things in QA a month after the commits were made, which makes it harder to fix those issues. Even if specific PRs don't get in, we routinely have dozens of commits across the product, even on a bi-weekly cycle. As things mature (and we create a 3.0.0 release someday(?)), this release cycle might slow down.

@denniskigen Since "bi-weekly" is ambiguous, maybe it's better to use "fortnightly"? (In my dialect, "fortnightly" is archaic, but it's at least unambiguous).